### PR TITLE
PHP 7.1 + remove XHProf/XDebug + remove vagrant + simplify docker

### PR DESCRIPTION
**Pull request by alexandresalome**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,87 +1,76 @@
...
 
 **or**
 
-  * PHP 5.3 or higher;
-  * PDO-SQLite PHP extension enabled;
-  * Xdebug
-  * XHProf
-  * [Blackfire](https://blackfire.io/getting-started)
-  * and the [usual Symfony application requirements](http://symfony.com/doc/current/reference/requirements.html).
+- PHP 7.1 or higher;
```
Comment (lolautruche) [inline README.md:15]: Let's require at least 7.2

File: README.md [hunk 2]
```diff
@@ -1,87 +1,76 @@
...
+And start server:
 
-### Usage
-
-First of all export your Blackfire environment variables:
 ```
-export BLACKFIRE_SERVER_ID=[YOUR-VALUE]
-export BLACKFIRE_SERVER_TOKEN=[YOUR-VALUE]
+# With embedded server
+app/console server:run --env=prod
```
Comment (lolautruche) [inline README.md:61]:
```suggestion
`app/console server:run --env=prod`
```

File: README.md [hunk 3]
```diff
@@ -1,87 +1,76 @@
...
+
+# Or with Symfony CLI
+symfony serve
```
Comment (lolautruche) [inline README.md:64]:
```suggestion
`symfony serve`
```

**Review by lolautruche — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alexandresalome) [reply README.md:61]: This code is inside a code-block:
    
    ```
    # With embedded server
    app/console server:run --env=prod

    # Or with Symfony CLI
    symfony serve
    ```

File: README.md (hunk 3 above)
Comment (alexandresalome) [reply README.md:64]: Same here

**Comment by alexandresalome** [comment]
Closed in favor of #7

**Pull request closed without merging**